packages/core/src/connectivity/scp-cf/destination/destination-service.ts

```diff
@@ -239,9 +238,7 @@ function callDestinationService(
   options: ResilienceOptions = { enableCircuitBreaker: true }
 ): Promise<HttpResponse> {
   const config: HttpRequestConfig = {
-    ...getAxiosConfigWithDefaults(),
-    url: uri,
-    method: 'GET',
+    method: 'get',
```

Comment:
The uri parameter is unused now, isn't it?

Reply:
yes, but it is part of the HttpRequestConfig.

Reply:
I mean the parameter in the function, `uri` - it is not used within the function, so you can remove it from the signature.

Reply:
it is used in line 245: ```  let destination: Destination = { url: uri, proxyType: 'Internet' };```

Reply:
Ah ok, then 👍